app/bundles/LeadBundle/Controller/ListController.php

```diff
@@ -74,8 +74,8 @@ public function indexAction($page = 1)
         $session->set('mautic.segment.filter', $search);
 
         //do some default filtering
-        $orderBy    = $session->get('mautic.segment.orderby', 'l.name');
-        $orderByDir = $session->get('mautic.segment.orderbydir', 'ASC');
+        $orderBy    = $session->get('mautic.segment.orderby', 'l.dateModified');
```

Comment:
I don't know If I would vote for change default ordering even If make sense
Customers used to use it for years, I am scare change it just by the time. 
But make sense this change

@escopecz @RCheesley what do you think?

Reply:
What about change this by local parameter? Should be something like this:

```php
$orderBy    = $this->get('session')->get('mautic.page.orderby',  $this->coreParametersHelper->get('default_pages_order', 'p.dateModified'));
```

It allow change it by local.php without BC. Everybody  can manage it for  customers or somebody can create configuration later.

What do you think?

Reply:
It would have to have the default order set to `l.name` to keep BC and whoever wants `l.dateModified` would have to go and set it in `local.php`. No user will do that unless they care enough. So this will become an unused feature.

@RCheesley what are your thoughts whether this is a BC change or not?

If it is then I'd rather start new 5.x branch and merge PRs like this one there.

Reply:
Honestly I think it would be more elegant if there were a setting where you could say 'sort by' and set your default preference in the global config, and have it apply everywhere, but I think that is a bit beyond the scope of this PR maybe.

To my mind this is not a BC break, it is a really small change in the user interface which is fixing a bug (we really should have been sorting by last modified). I think that if we have significant outcry, we can look at having some kind of global setting, but like @escopecz says above, I feel that almost everyone using Mautic will be sorting to find the stuff they used most recently, because of the way we have our interface set being counter-intuitive, so they will probably welcome the improvement. For me it is one of those annoyances that never bothered me enough to do anything about but I always have to work around!

Reply:
@escopecz it was example from Pages, I take it from another notes. 

Second concern for me it's about random order by for other section, somewhere is name, somewhere is modified date etc. There is no make sense

Now:

- email - by subject DESC
- pages - by title DESC
- assets - title DESC
- segmetns - name  ASC	
- forms - name ASC
- DC  - name DESC 
- Campaigns - name ASC 


Now we have few PRs add columns and just 3 of them (assets, pages, segments) change order by 

- https://github.com/mautic/mautic/pull/9691
- https://github.com/mautic/mautic/pull/9688
- https://github.com/mautic/mautic/pull/9685
- https://github.com/mautic/mautic/pull/9690

Just saying it add more confusion to that. 
Just don't understand why just some entities, and not others. 
Why we cannot do it unify or configurable by local.php at least.
Now it's not perfect, but this touch does not make sense for it.

Reply:
I see we've already change order by for campaigns  in previous version.

Reply:
I'm OK with the configuration. But you should have 2 config values per entity. One for the order by column and another for order by direction. When you consider how many entities there are in Mautic then the amount of config options will grow quite a lot. Then it will need a UI to make it actually useful for users. Then every time you install a new Mautic instance you'll have to go to the configuration and set the ordering to all entities from name ASC to date modified DESC. Of course it would be nice to have but is it worth the effort? Is this the priority considering everything else that's going on? Or shall we just set it to the default option that is useful for the majority and move on?

I also think the order should be unified across all entities, but I'm OK if it will be done in a span of several releases.